### ISSUE TITLE
Failing to only load compiled Typescript (.js) files in migrations directory

### ISSUE THREAD

**Issue by E-Kuerschner** [tgriesser/knex#3027]
# Environment
Node v8.1.4
Typescript v3.1.6

Knex version: 0.16.3
Database + version: Microsoft SQL Server 2017 (RTM-CU3-GDR) (KB4052987) - 14.0.3015.40 (X64) 
OS: macOS 10.14.2 (18C54)

# Bug
*Disclaimer*: This could be a duplicate of #2756 

`knex migrate:latest` fails on an invalid syntax error `SyntaxError: Unexpected token import` when it attempts to load `database/migrations/20180405225010_create_achievement_groups.ts`. We compile all our project's JS alongside the corresponding Typescript files. In previous Knex versions (v0.15.X) we utilized the `loadExtensions` migration config to instruct knex to filter out all but `.js`  migration files. 

Upon inspecting the source code I believe the `FsMigrations` class is not instantiated correctly.

(See line 20 of `sources/fs-migration.js` - notice the `loadExtensions` arg)
![image](https://user-images.githubusercontent.com/10578578/52183895-188cb200-27d2-11e9-8031-2c12ea547cc6.png)

(and then here on line 427 of `Migrator.js` - the constructor of `FsMigrations` is not called with the loadExtensions config)
![image](https://user-images.githubusercontent.com/10578578/52183910-4d006e00-27d2-11e9-9c71-d247e5de8658.png)

**Comment by E-Kuerschner** [comment]
As I started the pull request for this, I noticed the changes I were about to make have already been merged into master... I guess I'm just waiting on a new release then?

**Comment by kibertoad** [comment]
You can try 0.16.4-next release
